RELEASE-NOTES.md

```diff
@@ -4,6 +4,38 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.02 release notes (2020-02-18)
+Packages released:
+
```

Comment:
Do we need to include the package version here? If so, the latest on https://www.npmjs.com/package/@patternfly/patternfly is 2.63.0.

Reply:
Yes, thank you!